test/Project.toml

```diff
@@ -1,13 +1,5 @@
-[compat]
-BSON = "0.3.3"
-CairoMakie = "0.6, 0.7, 0.8, 0.9, 0.10"
-Flux = "0.13 - 0.13.12" # TODO: Return to "0.13" once https://github.com/FluxML/Flux.jl/issues/2204 is resolved
```

Comment:
Is this comment resolved already? If not can we keep it in the new test/Project.toml?

Reply:
Yes, this comment is resolved by updating the compat bounds to what it is now